unity-src/scripts/ZDKHelpCenter.cs

```diff
@@ -176,10 +176,9 @@ public class HelpCenterOptions {
 		}
 
 		/// <summary>
-		/// Displays a specific article. In Android it does this by using the the id of the article.
-		/// In iOS it does this by using the article json returned from a callback to get articles.
+		/// Pass the article ID
```

Comment:
Can you keep the "Displays a specific article" bit? `Pass the article ID` doesn't really say what the method does.

Reply:
Done